src/main/java/org/apache/mesos/dcos/DcosCluster.java

```diff
@@ -0,0 +1,33 @@
+package org.apache.mesos.dcos;
+
+import org.apache.http.client.fluent.Content;
+import org.apache.http.client.fluent.Request;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * Instances of this class represent DC/OS clusters.
+ */
+public class DcosCluster {
+    private URI dcosUri;
+    private static final String DCOS_VERSION_PATH = "/dcos-metadata/dcos-version.json";
+
+    public DcosCluster(URI dcosUri) {
```

Comment:
Assuming DcosCluster is constructed by Capabilities, this constructor could be package-private